docs/administrator-guide/installation-and-activation.md

```diff
@@ -0,0 +1,16 @@
+## Installing Channel Export for Self-Managed Deployments
+
+Mattermost Incident Management is available in the Plugin Marketplace. You can download and install the plugin from Mattermost.
+
+1. Open **System Console > Plugin Management**.
```

Comment:
We should clarify that this step to enable the plugin comes after already having installed the plugin from the marketplace. I was confused reading this as the first step.

Reply:
Good point, thanks.

Reply:
@itao, confirming that this has been addressed.